scripts/tests/test_node_summary.py

```diff
@@ -56,21 +44,24 @@ def setUp(self):
         self.deleted_node = ProjectFactory(is_deleted=True)
         self.deleted_node2 = ProjectFactory(is_deleted=True)
 
-        self.date = datetime.datetime.utcnow() - datetime.timedelta(1)
+        self.date = timezone.now() - datetime.timedelta(1)
 
-        modify_node_dates_in_mongo(self.date - datetime.timedelta(0.1))
+        for node in AbstractNode.objects.all():
+            node.date_created = self.date - datetime.timedelta(0.1)
+            node.save()
+        # modify_node_dates_in_mongo(self.date - datetime.timedelta(0.1))
 
         self.results = NodeSummary().get_events(self.date.date())[0]
 
     def tearDown(self):
         super(TestNodeCount, self).tearDown()
-        Node.remove()
-        User.remove()
+        AbstractNode.remove()
+        OSFUser.remove()
 
     def test_get_node_count(self):
         nodes = self.results['nodes']
 
-        assert_equal(nodes['total'], 8)  # 3 Projects, 5 Registrations
+        assert_equal(nodes['total'], 3)  # 2 Projects, 1 component
```

Comment:
This was a mistake! Nodes should never have been counting registrations as well, so the 8 was including registrations. 3 is what the number should have been without registrations.